ilastik/utility/slot_name_enum.py

```diff
@@ -0,0 +1,49 @@
+import enum
+from itertools import chain
+
+class SlotNameEnum(enum.IntEnum):
+    @property
+    def socketName(self):
```

Comment:
The reason for 'socket' instead of 'slot' is because I'm an idiot ^^
For some reason I've mapped the 'slot' concept to the word 'socket' -.-
It should absolutely be "slot". I'll fix that. I'll also add some docstrings around.